### PR TITLE
added today func, test and docs

### PR DESCRIPTION
**Pull request by shubham-shay-te**

### CODE REVIEW

File: tests/test_datetime_utils.py [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
 from datetime import datetime, timedelta
 
-from dateutil.utils import today
-
 from core_lib.helpers.datetime_utils import year_begin, year_end, month_begin, month_end, week_begin, week_end, \
-    day_begin, day_end, tomorrow, yesterday, midnight, sunday, monday, tuesday, wednesday, thursday, friday, saturday, \
-    hour_begin, hour_end
+    day_begin, day_end, tomorrow, today, yesterday, midnight, sunday, monday, tuesday, wednesday, thursday, friday, \
+    saturday, hour_begin, hour_end
 from core_lib.helpers.func_utils import reset_datetime
```
Comment (shay-te) [inline tests/test_datetime_utils.py:8]: where is the test for the `today` function?

**Review by shay-te — CHANGES_REQUESTED** [review]

File: tests/test_datetime_utils.py (hunk 1 above)
Comment (shubham-shay-te) [reply tests/test_datetime_utils.py:8]: the test was already written but it had imported today() `from dateutil.utils import today` from here, so while adding hints i noticed that

Comment (shay-te) [reply tests/test_datetime_utils.py:8]: thank you. didn't follow. do we have a today tesT?

File: core_lib/helpers/datetime_utils.py [hunk 2]
```diff
@@ -44,17 +44,21 @@ def day_end(hours: int = 0, minutes: int = 0):
```
Comment (shay-te) [inline core_lib/helpers/datetime_utils.py:44]: where is the new today function?

Comment (shubham-shay-te) [reply core_lib/helpers/datetime_utils.py:44]: it's on line 51

File: tests/test_datetime_utils.py (hunk 1 above)
Comment (shay-te) [reply tests/test_datetime_utils.py:8]: please can we add test. to  assert `from dateutil.utils import today` with core-lib `today function`
thank you

**Review by shay-te — CHANGES_REQUESTED** [review]

File: tests/test_datetime_utils.py [hunk 3]
```diff
@@ -47,14 +49,14 @@ def test_yesterday(self):
         self.assertEqual(cl_yesterday.year, dt_yesterday.year)
 
     def test_today(self):
-        dt_today = datetime.today()
+        dtu_today = dateutils_today()
         cl_today = today()
         self.assertNotEqual(cl_today, None)
         self.assertEqual(cl_today.minute, 0)
         self.assertEqual(cl_today.hour, 0)
```
Comment (shay-te) [inline tests/test_datetime_utils.py:56]: what is the minutes, hour. of  dtu_today?

**Review by shay-te — APPROVED** [review]
Thank you good work.

File: tests/test_datetime_utils.py (hunk 3 above)
Comment (shubham-shay-te) [reply tests/test_datetime_utils.py:56]: added the test to test with `dtu_today`'s minute and hour

Comment (shubham-shay-te) [reply tests/test_datetime_utils.py]: it's returning `00:00:00`

**Pull request merged**